Common/include/geometry/CGeometry.hpp

```diff
@@ -293,20 +293,24 @@ class CGeometry {
    * \param[in] geometry - Geometrical definition of the problem.
    * \param[in] config   - Definition of the particular problem.
    * \param[in] commType - Enumerated type for the quantity to be communicated.
-   * \param[in] val_reverse  - Boolean controlling forward or reverse communication between neighbors.
+   * \param[in] countPerPoint - Number of variables per point.
+   * \param[in] val_reverse - Boolean controlling forward or reverse communication between neighbors.
    */
-  void PostP2PRecvs(CGeometry *geometry, CConfig *config, unsigned short commType, bool val_reverse) const;
+  void PostP2PRecvs(CGeometry *geometry, CConfig *config, unsigned short commType,
+                    unsigned short countPerPoint, bool val_reverse) const;
```

Comment:
A consequence of this is that the current count-per-point needs to be explicitly passed to a few routines, whereas before we could always rely on the maximum value.